src/main/java/org/rumbledb/config/SparksoniqRuntimeConfiguration.java

```diff
@@ -62,6 +62,13 @@ public boolean getOverwrite() {
             return false;
     }
 
+    public boolean getErrorInfo() {
+        if (this._arguments.containsKey("error-info"))
+            return this._arguments.get("error-info").equals("yes");
+        else
+            return false;
+    }
```

Comment:
How would you feel about the name 'show-error-info' for flag/configuration. The getter method naming is a bit cryptic as it is. The current use case of the configuration and its naming in Main.java (I will also tag this), felt misleading to me

Reply:
Yes, I agree. This is done.